src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedMathUtils.java

```diff
@@ -189,6 +189,27 @@ public static double stdDev(final double ... values) {
         return new EnumeratedDistribution<>(rng, pmf).sample();
     }
 
+/** Compute Z=X+Y for two numeric vectors X and Y
+     *
+     * @param x                 First vector
+     * @param y                 Second vector
+     * @return Vector of same length as x and y so that z[k] = x[k]+y[k]
+     */
+    public static double[] vectorSum(final double[]x, final double[] y) {
```

Comment:
actually, where is this method used?  if we keep it, probably should add a simple unit test to be safe